test/bdd/features/introduce.feature

```diff
@@ -10,284 +10,177 @@ Feature: Introduce protocol
 
     When   "Bob" sends introduce request to the "Alice" asking about "Carol"
-      And   "Alice" sends introduce proposal back to the "Bob" and requested introduce
-      And   "Bob" wants to know "Carol" and sends introduce response with approve
-
-    When   "Alice" continue with the introduce protocol
-      And   "Carol" doesn't want to know "Bob" and sends introduce response
-      And   "Alice" continue with the introduce protocol
+    Then   "Alice" sends introduce proposal back to the "Bob" and requested introduce
+    And   "Bob" wants to know "Carol" and sends introduce response with approve
+    And   "Carol" doesn't want to know "Bob" and sends introduce response
 
-    Then   "Alice" checks the history of introduce protocol events "arranging,arranging,arranging,arranging,delivering,delivering,abandoning,abandoning,done,done"
+    Then   "Alice" checks the history of introduce protocol events "arranging,arranging,arranging,arranging,arranging,arranging,abandoning,abandoning,done,done"
     And   "Bob" checks the history of introduce protocol events "requesting,requesting,deciding,deciding,waiting,waiting,abandoning,abandoning,done,done"
-    And   "Carol" checks the history of introduce protocol events "deciding,deciding,abandoning,abandoning,done,done"
-
-  @proposal_introducer_stop
-  Scenario: Introducer stops the protocol after receiving approve
-    Given   "Bob,Carol" exchange DIDs with "Alice"
-
-    When  "Bob" creates introduce client with an invitation
-      And   "Alice" sends introduce proposal to the "Bob" and "Carol"
-      And   "Bob" wants to know "Carol" and sends introduce response with approve
-      And   "Alice" stops the introduce protocol
-
-    Then   "Alice" checks the history of introduce protocol events "arranging,arranging,abandoning,abandoning,done,done"
-      And   "Bob" checks the history of introduce protocol events "deciding,deciding,waiting,waiting,abandoning,abandoning,done,done"
+    And   "Carol" checks the history of introduce protocol events "deciding,deciding,abandoning,abandoning,done,done" and stop
 
   @proposal_introducer_stop_with_request
-  Scenario: Introducer stops the protocol after receiving approve the protocol starts with request
-    Given   "Bob,Carol" exchange DIDs with "Alice"
-
-    When   "Bob" creates introduce client with an invitation
-      And   "Bob" sends introduce request to the "Alice" asking about "Carol"
-      And   "Alice" sends introduce proposal back to the "Bob" and requested introduce
-      And   "Bob" wants to know "Carol" and sends introduce response with approve
-      And   "Alice" stops the introduce protocol
-
-    Then   "Alice" checks the history of introduce protocol events "arranging,arranging,abandoning,abandoning,done,done"
-    And   "Bob" checks the history of introduce protocol events "requesting,requesting,deciding,deciding,waiting,waiting,abandoning,abandoning,done,done"
-
-  @proposal_introducer_second_stop
```

Comment:
is this case no longer relevant?

Reply:
Same. Related to the tests which contains `introducer..stop`